torch/csrc/distributed/autograd/context/dist_autograd_container.cpp

```diff
@@ -113,6 +114,11 @@ bool DistAutogradContainer::hasValidContext() const {
   return current_context_id_ != kInvalidContextId;
 }
 
+bool DistAutogradContainer::hasContextWithId(int64_t context_id) const {
```

Comment:
nit: `hasContextId`

Reply:
This will actually go away since we are checking + releasing atomically with `releaseContextIfPresent`